lsp-ui-flycheck.el

```diff
@@ -208,7 +209,10 @@ See https://github.com/emacs-lsp/lsp-mode."
   :start #'lsp-ui-flycheck--start
   :modes '(python-mode) ; Need a default mode
   :predicate (lambda () lsp-mode)
-  :error-explainer (lambda (e) (flycheck-error-message e)))
+  :error-explainer (lambda (e)
```

Comment:
We have to figure out where to put this logic - ATM I am moving the flycheck integration in lsp-mode. Is it possible to alter the flycheck checker once it is defined?

Reply:
I think so, but I'd need to check the Flycheck API. Where do you want to move the flycheck integration?

Reply:
What do you think about having `lsp-clangd.el` in lsp-mode to host this functionality? Later we could use the same code for `lsp-treemacs-errors-list`.

Reply:
we may keep it as it is and use `functionp` to access this function if it is present. Of course, I am open to alternative proposals.

Reply:
I'll open a PR against the `lsp-mode` repo, now that the logic has moved.